docs/contributions/Contributions.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+title: How to Contribute FAQ Guidelines
```

Comment:
```suggestion
title: Contributing
```

Excuse the original unnecessarily verbose title suggestion. I would also change the file name to `Contributing.md`